src/export_edf.jl

```diff
@@ -96,18 +96,90 @@ function onda_samples_to_edf_header(samples::AbstractVector{<:Samples};
                           is_contiguous, edf_record_metadata(samples)...)
 end
 
+"""
+    reencode_samples(samples::Samples, sample_type::Type{<:Integer}=Int16)
+
+Encode `samples` so that they can be encoded as `sample_type`.  The default
```

Comment:
```suggestion
Encode `samples` so that they can be stored as `sample_type`.  The default
```

the double `encode` feels weird